newIDE/app/src/AssetStore/ExampleStore/ExampleIcon.js

```diff
@@ -18,19 +18,29 @@ const styles = {
 
 type Props = {|
   exampleShortHeader: ExampleShortHeader,
-  size: number,
+  type: 'thumbnail' | 'preview',
 |};
 
-export const ExampleIcon = ({ exampleShortHeader, size }: Props) => {
+export const ExampleIcon = ({ exampleShortHeader, type }: Props) => {
+  const size = type === 'thumbnail' ? 40 : 120;
+  let iconUrl = exampleShortHeader.previewImageUrls[0];
```

Comment:
What if the list is empty?

Reply:
this should be handled:
- there's always a list
- if it's empty, this will return null/undefined
- the CorsAwareImage component knows how to handle this and returns undefined